### PR TITLE
feature/120_pr_template

### PR DESCRIPTION
**Pull request by damian-romero**
Before merging, determine which of the templates are useful or if we want to go with the generic vs bump-version uption. The following checklist is to decide which templates we'll keep:

- [x] PULL_REQUEST_TEMPLATE/bug-pull-request.md
- [x] PULL_REQUEST_TEMPLATE/feature-pull-request.md
- [ ] PULL_REQUEST_TEMPLATE/generic-pull-request.md
- [x] PULL_REQUEST_TEMPLATE/release-pull-request.md

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<!-- Explain how you tested this big-fix so that others can replicate it -->
+<!--  Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI. -->
+
+**Bump version information**
+Bump version: v0.xx.xx -> v0.xx.x
+
+Bumped in:
+- [ ] README.md, [installation instructions](https://github.com/ualibraries/ReQUIAM#installation-instructions)
+- [ ] [setup.py](https://github.com/ualibraries/ReQUIAM/blob/master/setup.py)
+- [ ] [requiam/__init__.py](https://github.com/ualibraries/ReQUIAM/blob/master/requiam/__init__.py)
```
Comment (damian-romero) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]: requiam/__init__.py should display as `requiam/__init__.py`
```suggestion
- [ ] [requiam/\_\_init\_\_.py](https://github.com/ualibraries/ReQUIAM/blob/master/requiam/__init__.py)
```

File: .github/PULL_REQUEST_TEMPLATE/feature-pull-request.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+**Approach**
+<!-- How does this change address the problem? -->
+
+<!-- Add any open questions and Pre-Merge TODOs. Use checkboxes. -->
+<!-- - [ ]   -->
+
+**Test plan (If applicable)**
+<!-- Explain how you tested this feature so that others can replicate it -->
+<!--  Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI. -->
```
Comment (damian-romero) [inline .github/PULL_REQUEST_TEMPLATE/feature-pull-request.md]:
```suggestion
<!-- Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI. -->
```

File: .github/PULL_REQUEST_TEMPLATE/release-pull-request.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+*Screenshots or additional context*
+<!-- Add any other context about the problem here and/or screenshots to help explain your problem. -->
+
+**Bump version information**
+Bump version: v0.xx.xx -> v0.xx.x
+
+Bumped in:
+- [ ] README.md, [installation instructions](https://github.com/ualibraries/ReQUIAM#installation-instructions)
+- [ ] [setup.py](https://github.com/ualibraries/ReQUIAM/blob/master/setup.py)
+- [ ] [requiam/__init__.py](https://github.com/ualibraries/ReQUIAM/blob/master/requiam/__init__.py)
```
Comment (damian-romero) [inline .github/PULL_REQUEST_TEMPLATE/release-pull-request.md]: requiam/__init__.py should display as `requiam/__init__.py`
```suggestion
- [ ] [requiam/\_\_init\_\_.py](https://github.com/ualibraries/ReQUIAM/blob/master/requiam/__init__.py)
```

**Review by damian-romero — COMMENTED** [review]
A couple of format changes were necessary.

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+name: Pull request - bug
+about: Provide specifics about a pull request for bugs
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
about: Provide specifics about a pull request for bug fixes
```

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+---
+<!-- IMPORTANT: Please do not create a Pull Request without creating an issue first. -->
+
+<!--Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL -->
+
+**Description**
+<!-- A clear and concise description of the problem the bug fix in this PR tries to solve -->
+
+<!-- Add a linked issues/pull requests -->
+See #
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
Fixes #
```

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+name: Pull request - bug
+about: Provide specifics about a pull request for bugs
+title: 'PR - bug: ____(provide a descriptive name)'
+labels: bug, p1
+assignees: ''
+
+---
+<!-- IMPORTANT: Please do not create a Pull Request without creating an issue first. -->
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
<!-- IMPORTANT: Please do not create a PR without creating an issue first. -->
```

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+<!--Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL -->
+
+**Description**
+<!-- A clear and concise description of the problem the bug fix in this PR tries to solve -->
+
+<!-- Add a linked issues/pull requests -->
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
<!-- Add any linked issue(s) -->
```

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+See #
+
+*Screenshots or additional context*
+<!-- Add any other context about the problem here and/or screenshots to help explain your problem. -->
+
+**Approach**
+<!-- How does this bug-fix address the problem? -->
+
+<!-- Add any open questions and Pre-Merge TODOs. Use checkboxes. -->
+<!-- - [ ]   -->
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
```

I rather have this information in the issue instead.

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+**Bump version information**
+Bump version: v0.xx.xx -> v0.xx.x
+
+Bumped in:
+- [ ] README.md, [installation instructions](https://github.com/ualibraries/ReQUIAM#installation-instructions)
+- [ ] [setup.py](https://github.com/ualibraries/ReQUIAM/blob/master/setup.py)
+- [ ] [requiam/__init__.py](https://github.com/ualibraries/ReQUIAM/blob/master/requiam/__init__.py)
+
+*Resources*
+<!-- Links to blog posts, StackOverflow, libraries or add-ons used to solve this problem -->
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
```

I prefer that this stays in the issue information.

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
+---
+name: Pull request - bug
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
name: Bug
```

This is the name that shows up under PR page. It's redundant as a result

File: .github/PULL_REQUEST_TEMPLATE/release-pull-request.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+<!--Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL -->
+
+**Description**
+<!-- A clear and concise description of the updates contained in this release. We usually use a list. -->
+
+Improvements include:
+- A #issue
+- B #issue
+- C #issue
+- n
```
Comment (astrochun) [inline .github/PULL_REQUEST_TEMPLATE/release-pull-request.md]: Not sure what you're trying to do here. For releases, we want to auto close issues.

**Review by astrochun — COMMENTED** [review]
Accepted @damian-romero suggestions

Additional changes incorporated.

In the future, start with a simple generic one so we can improve. A typo in one results in four typos to fix.

File: .github/PULL_REQUEST_TEMPLATE/release-pull-request.md (hunk 11 above)
Comment (damian-romero) [inline .github/PULL_REQUEST_TEMPLATE/release-pull-request.md]:
```suggestion
<!-- A clear and concise description of the updates contained in this release. Example:  -->
<!-- Incorporates Travis CI feature #91, summary for script_run #99, and root portal handling #88 -->

- Fixes #
- Fixes #
- Fixes #
- Fixes #
```

File: .github/PULL_REQUEST_TEMPLATE/feature-pull-request.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
+---
+name: Pull request - feature
```
Comment (damian-romero) [inline .github/PULL_REQUEST_TEMPLATE/feature-pull-request.md]:
```suggestion
Feature
```

File: .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+labels: bug, p1
+assignees: ''
+
+---
+<!-- IMPORTANT: Please do not create a PR without creating an issue first. -->
+
+<!--Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL -->
+
+**Description**
+<!-- A clear and concise description of the problem the bug fix in this PR tries to solve -->
```
Comment (damian-romero) [inline .github/PULL_REQUEST_TEMPLATE/bug-pull-request.md]:
```suggestion
<!-- A clear and concise description of the problem the bug that this PR tries to solve -->
```

**Comment by damian-romero** [comment]
@astrochun the templates are ready for your review.

**Comment by astrochun** [comment]
@damian-romero I've implemented a few changes to clean it up. You can view how it will look [here](https://github.com/ualibraries/ReQUIAM/tree/feature/120_pr_template/.github/PULL_REQUEST_TEMPLATE)

**Review by damian-romero — COMMENTED** [review]
I think they're set

**Pull request merged**